cpp/dolfinx/mesh/graphbuild.h

```diff
@@ -52,12 +55,12 @@ build_local_dual_graph(CellType celltype, std::span<const std::int64_t> cells);
 /// @note Collective function
 ///
 /// @param[in] comm The MPI communicator
-/// @param[in] celltype The cell type
-/// @param[in] cells Collection of cells, defined by the cell vertices
-/// from which to build the dual graph
+/// @param[in] celltypes List of cell types
```

Comment:
Precondition is `celltypes.size() == cells.size()`?

Docstring doesn't explain relationship between `celltypes` and cells`.